README.md

```diff
@@ -35,8 +35,7 @@ View, Controller**.
 
 We're going to talk about MVC, because that's the pattern that Rails implements.
 
-You've seen Sinatra so you've already seen a pattern that is very close to MVC,
-but not quite. ActiveRecord has already shown you a library that helps you build *models*.
+We've already used the MVC design paradigm to organize our Sinatra apps. ActiveRecord has already shown you a library that helps you build *models*. We've already used Embedded Ruby (erb) to construct *views*.
```

Comment:
Maybe the latter two sentences can be bulletpoints under the first sentence. Otherwise it reads sort of run-on.